ofl/kodemono/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The Kode Mono Project Authors (https://github.com/isaozler/kode-mono)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://scripts.sil.org/OFL
```

Comment:
OFL url is no longer "https://scripts.sil.org/OFL". Use 'https://openfontlicense.org/' instead.